docfx.json

```diff
@@ -203,14 +203,11 @@
       "brand": "azure",
       "author": "rloutlaw",
       "ms.author": "routlaw",
```

Comment:
manager is required metadata. I believe that Barb Kess it now the manager that owns this content.